sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.3.0-beta.1 (Unreleased)
 
 ### Features Added
+- This version of the client library marks a preview release and defaults to the service API version `2022-10-01-preview`.
```

Comment:
nitpick: The SDK uses beta rather than preview
```suggestion
- This version of the client library marks a beta release and defaults to the service API version `2022-10-01-preview`.
```